pipelines/LSSTComCamSim/cpCti.yaml

```diff
@@ -0,0 +1,4 @@
+description: LSSTCam cti calibration construction
+instrument: lsst.obs.lsst.LsstComCamSim
+imports:
+  - location: $CP_PIPE_DIR/pipelines/_ingredients/cpCtiLSST.yaml
```

Comment:
This is fine such as it is, but note that LSSTComCamSim doesn't have any CTI to my knowledge.